### PR TITLE
Make Date serialization follow Date Time String format

### LINKED ISSUE #202: Unify Date serialization and deserialization
Currently:
* [serialization](https://w3c.github.io/webdriver-bidi/#data-types-protocolValue-RemoteValue-serialization) specifies the Date value should be a result of [Date.ToDateString](https://tc39.es/ecma262/#sec-todatestring)([thisTimeValue](https://tc39.es/ecma262/#thistimevalue)(value)), which ends up with a string like `Thu May 05 2000 15:12:53 GMT+0200 (Central European Summer Time)`.
* [deserialization](https://w3c.github.io/webdriver-bidi/#data-types-protocolValue-LocalValue-deserialization) specifies a check if the value matches [Date Time String Format](https://tc39.es/ecma262/#sec-date-time-string-format) "ISO 8601 calendar date extended format", which look like `YYYY-MM-DDTHH:mm:ss.sssZ`.

I suggest to make serialized Date value to be "ISO 8601 calendar date extended format".

### PR DESCRIPTION
**Pull request by lutien**
Closes #202

***
<a href="https://pr-preview.s3.amazonaws.com/lutien/webdriver-bidi/pull/237.html" title="Last updated on Jun 20, 2022, 7:37 AM UTC (d874621)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webdriver-bidi/237/7e81a9e...lutien:d874621.html" title="Last updated on Jun 20, 2022, 7:37 AM UTC (d874621)">Diff</a>

### CODE REVIEW

**Comment by whimboo** [comment]
CC'ing @sadym-chromium in case he has some feedback. If not the change looks small enough so that we can get it merged. Will do so in about 1-2h.

File: index.bs [hunk 1]
```diff
@@ -1744,7 +1744,9 @@ an |ownership type|, a |serialization internal map| and a |realm|:
 
     <dt>|value| has a \[[DateValue]] [=internal slot=].
     <dd>
-      1. Let |serialized| be [=Date.ToDateString=]([=thisTimeValue=](|value|)).
+      1. Set |serialized| to [=Call=]([=Date.toISOString=], |value|).
```
Comment (sadym-chromium) [inline index.bs:1747]: As I understand, `toISOString` makes the time UTC regardless of the timezone. Meaning serialization loses part of the information of the original ECMAScript value. Not sure how critical is that though.

Comment (jgraham) [reply index.bs:1747]: I don't think that ECMAScript Date values store a timezone; from https://tc39.es/ecma262/#sec-time-values-and-time-range they are just a unix timestamp internally.

Comment (sadym-chromium) [reply index.bs:1747]: ok, not an issue.

**Pull request merged**